NOTICE

```diff
@@ -1,7 +1,7 @@
 Copyright 2019 Jehan Tremback
 Copyright 2019-2021 Confio OÜ
 Copyright 2019-2020 Simon Warta
-Copyright 2021 Confio GmbH
+Copyright 2021,2022,2023,2024 Confio GmbH
```

Comment:
Maybe `2021-2024` would be simpler?

Reply:
Yeah, no strong opinion. I once heard a guy saying it is better to list the years of actual authorship such that years without contributions are skipped. This is why I did it that way.

Reply:
Yeah, that makes sense, but I don't see us stopping for a year inbetween. :D
And then the list will get very long (hopefully).

Reply:
But any way is fine for me. We can still switch if it gets out of hand.